examples/apps/meshdemo/kubernetes/app/kustomize/base/cats.yaml

```diff
@@ -0,0 +1,134 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: cats
+  labels:
+    name: cats
+  namespace: meshdemo
+spec:
+  ports:
+  - port: 8080
+  selector:
+    name: cats
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: cats
+  namespace: meshdemo
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        name: cats
+    spec:
+      serviceAccountName: xray-access
+      containers:
+      - name: cats
+        image: 072792469044.dkr.ecr.eu-west-1.amazonaws.com/meshdemo/cats:1.0.0
```

Comment:
will these be replaced by kustomize?

Reply:
I am open to hosting these images wherever.  Currently they are on my AWS internal account, but a quick change to the Makefile and send them anywhere, including docker hub.